rust/lance-encoding/src/encodings/physical/bitpack.rs

```diff
@@ -579,6 +579,7 @@ fn rows_in_buffer(
     bits_in_buffer / bits_per_value
 }
 
+/*
```

Comment:
I think I fixed this (well, more worked around it).  Can you uncomment and verify these tests still fail?